diff --git a/wew.c b/wew.c
index 74dd417..35a68e6 100644
--- a/wew.c
+++ b/wew.c
@@ -12,7 +12,7 @@ static xcb_screen_t *scr;
 static uint32_t mask = XCB_EVENT_MASK_NO_EVENT
                      /* | XCB_EVENT_MASK_KEY_PRESS */
                      /* | XCB_EVENT_MASK_KEY_RELEASE */
-                     /* | XCB_EVENT_MASK_BUTTON_PRESS */
+                     | XCB_EVENT_MASK_BUTTON_PRESS
                      /* | XCB_EVENT_MASK_BUTTON_RELEASE */
                      | XCB_EVENT_MASK_ENTER_WINDOW
                      /* | XCB_EVENT_MASK_LEAVE_WINDOW */
@@ -204,7 +204,7 @@ handle_events(void)
 		                XCB_EVENT_MASK_BUTTON_PRESS |
 		                XCB_EVENT_MASK_BUTTON_RELEASE,
 		                XCB_GRAB_MODE_ASYNC, XCB_GRAB_MODE_ASYNC,
-		                scr->root, XCB_NONE, 1, XCB_NONE);
+		                scr->root, XCB_NONE, 2, XCB_NONE);
 	}
 
 	/* register the events on all mapped windows */
